bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -861,21 +868,28 @@ impl Connection {
         let handshake = unsafe {
             s2n_connection_get_handshake_type_name(self.connection.as_ptr()).into_result()?
         };
-        // The strings returned by s2n_connection_get_handshake_type_name
-        // are static and immutable after they are first calculated
-        static_const_str!(handshake)
+        // SAFETY: constructed strings have a null byte appended to them
+        // SAFETY: the data has a 'static lifetime, because it resides in a static
+        // char array, and is never modified after it's initial creation
+        unsafe { const_str!(handshake) }
     }
 
     pub fn cipher_suite(&self) -> Result<&str, Error> {
         let cipher = unsafe { s2n_connection_get_cipher(self.connection.as_ptr()).into_result()? };
-        // The strings returned by s2n_connection_get_cipher
-        // are static and immutable since they are const fields on static const structs
-        static_const_str!(cipher)
+        // SAFETY: The data is null terminated because it is declared as a C string literal.
+        // SAFETY: The data has a static lifetime because it is associated with 
```

Comment:
```suggestion
        // SAFETY: cipher has a static lifetime because it lives on s2n_cipher_suite, a static struct.
```
I think it's more clear if you reference exactly which struct you're referring to (which I assume is s2n_cipher_suite). Also cipher_suites aren't const structs.